guide/src/class.md

```diff
@@ -181,6 +182,59 @@ created from Rust, but not from Python.
 
 For arguments, see the [`Method arguments`](#method-arguments) section below.
 
+## Initializer
+
+An initializer implements Python's `__init__` method.
```

Comment:
We should discourage users from using `init` unless they really need it; they probably should use `new` instead.